.github/workflows/release.yml

```diff
@@ -17,13 +17,35 @@ jobs:
       - id: release
         uses: google-github-actions/release-please-action@v4
 
+
       # If release-please created a release, publish to npm
-      - if: ${{ steps.release.outputs.release_created }}
+      - if: steps.release.outputs.release_created
```

Comment:
Is this correct? I thought you needed the ${{ }} to insert the value, otherwise this would just be the string "steps.release.outputs.release_created" no?

Reply:
Also, just an optional improvement, but since we're going to be duping this everywhere I think it is worth doing. We can make the release-please it's own "job" that outputs this, and then this job "needs" release-please. Then the whole job can be conditional upon "release_created" rather than every step having to declare this.

Reply:
I've separated into two jobs, thanks!